packages/react-core/src/components/Badge/__tests__/Badge.test.tsx

```diff
@@ -32,6 +32,17 @@ test('Renders with class name pf-m-read when isRead prop is true', () => {
   expect(screen.getByText('Test')).toHaveClass('pf-m-read');
 });
 
+test('Renders with toggle styles and icon when isToggle prop is true', () => {
+  render(
+    <Badge isRead={true} isToggle={true}>
+      Test
+    </Badge>
+  );
+  expect(screen.getByText('Test').lastChild).toContainHTML(
+    '<span class="pf-v5-c-badge__toggle-icon"><i class="fas fa-caret-down" /></span>'
+  );
+});
```

Comment:
Based on other tests we've done I might expect this test just to check that the toggle icon class is present on the element.

Reply:
done, ty!